lib/yandex/disk/client/request/private.rb

```diff
@@ -0,0 +1,22 @@
+class Yandex::Disk::Client::Request::Private
+  BODY =
```

Comment:
Maybe use `freeze` and `make_private` everywhere? I've decided not to add it to the pull request because of consistency with exists code. What do you think?

Reply:
We can do freeze in separate pr, yep.